lib/db_utils.go

```diff
@@ -11915,6 +11916,95 @@ func _enumerateKeysForPrefixWithLimitOffsetOrderWithTxn(
 	return keysFound, valsFound, nil
 }
 
+func EnumerateKeysOnlyForPrefixWithLimitOffsetOrderAndSkipFunc(
```

Comment:
Do you just want to replace the old `EnumerateKeysForPrefixWithLimitOffsetOrderAndSkipFunc` with this function? The old one is dead code now

Reply:
I do like having the name change to `KeysOnly` tho. will just delete the old one.